jest.config.js

```diff
@@ -16,6 +16,9 @@ module.exports = {
   setupFiles: [
     '<rootDir>/tests/setup.js'
   ],
+  snapshotSerializers: [
+    "enzyme-to-json/serializer"
+  ],
```

Comment:
How can we see the generated snapshot?

Reply:
When you run the test, check the snapshot files generated.
They should look really cool just like you have in you chrome elements-inspection. 😄